lib/style-transform.js

```diff
@@ -1,156 +1,85 @@
 const stylis = require('stylis')
 
-/**
- * @type {RegExp}
- *
- * matches :global($1) capturing $1
- */
-const regExpReplaceGlobal = /:global\((.*)\)/g
+function disableNestingPlugin(...args) {
+  let [context, , , parent = [], line, column] = args
```

Comment:
@giuseppeg Is there a reason to avoid named arguments? Babel will produce a bit more code for this [http://babeljs.io/repl/#?...](http://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&targets=&browsers=&builtIns=false&debug=false&experimental=false&loose=false&spec=false&code=function%20disableNestingPlugin(...args)%20%7B%0A%20%20let%20%5Bcontext%2C%20%2C%20%2C%20parent%20%3D%20%5B%5D%2C%20line%2C%20column%5D%20%3D%20args%0A%20%20if%20(context%20%3D%3D%3D%202)%20%7B%0A%20%20%20%20parent%20%3D%20parent%5B0%5D%0A%20%20%20%20if%20(typeof%20parent%20%3D%3D%3D%20'string'%20%26%26%20parent.trim().length%20%3E%200)%20%7B%0A%20%20%20%20%20%20throw%20new%20Error(%0A%20%20%20%20%20%20%20%20%60Nesting%20detected%20at%20%24%7Bline%7D%3A%24%7Bcolumn%7D.%20%60%20%2B%0A%20%20%20%20%20%20%20%20%20%20'Unfortunately%20nesting%20is%20not%20supported%20by%20styled-jsx.'%0A%20%20%20%20%20%20)%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D&playground=true)